VideosController.js

```diff
@@ -0,0 +1,44 @@
+
+var express = require('express');
+var router = express.Router();
+var Events = require('./Events');
+
+router.get('/', function (req, res) {
+    Events.find({
+        $or: [{event: "StreamCreated"}, {event: "StreamEnded"}]
+    }).sort({
+        createdAt: 1
+    }).exec(function (err, events) {
+        if (err) return res.status(500).send("There was a problem finding the video.")
+
+        videos = {};
+        for (let i of events) {
+            event = events[i]
```

Comment:
Sorry I think I needed to some coffee earlier. You can actually use `for (let event of events) {...}` and skip the `event = events[i]` altogether since `for...of` iterates over values of each entries rather than indexes

Reply:
👍 @jozanza what do you suggest for unit tests here?  What tool and how should we set it up nicely like you did for `livepeerjs`?